patches/chrome-android-java-src-org-chromium-chrome-browser-toolbar-ToolbarManager.java.patch

```diff
@@ -11,6 +11,15 @@ index f47d98236abb257f3a87e44c216f15bec6928b3c..bb361c1df24d7529e97aa5d516d55dc1
                  mControlsVisibilityDelegate, mActivity,
                  (focus, type)
                          -> setUrlBarFocus(focus, type),
+@@ -313,7 +313,7 @@ public class ToolbarManager implements UrlFocusChangeListener, ThemeColorObserve
+                 mActivity.isCustomTab() ? mCustomTabThemeColorProvider
+                                         : browsingModeThemeColorProvider,
+                 R.id.menu_button_wrapper);
+-        MenuButtonCoordinator startSurfaceMenuButtonCoordinator = new MenuButtonCoordinator(
++        MenuButtonCoordinator startSurfaceMenuButtonCoordinator = new org.chromium.chrome.browser.toolbar.menu_button.BraveMenuButtonCoordinator(
```

Comment:
change class with asm

Reply:
It requires additional fix in bytecode, issue created https://github.com/brave/brave-browser/issues/12284